tests/unit/draggable/core.js

```diff
@@ -338,6 +338,27 @@ QUnit.test( "blur behavior - descendant of handle", function( assert ) {
 	} );
 } );
 
+QUnit.test( "blur behavior - off handle", function( assert ) {
+	var ready = assert.async();
+	assert.expect( 3 );
+
+	var element = $( "#draggable2" ).draggable( { handle: "span" } ),
+		focusElement = $( "<div tabindex='1'></div>" ).appendTo( element );
+
+	testHelper.onFocus( focusElement, function() {
+		assert.strictEqual( document.activeElement, focusElement.get( 0 ), "test element is focused before mousing down on a draggable" );
+
+		testHelper.move( focusElement, 1, 1 );
+
+		assert.strictEqual( document.activeElement, focusElement.get( 0 ), "test element is focused after mousing down on itself" );
+
+		testHelper.move( element, 1, 1 );
+
+		assert.strictEqual( document.activeElement, focusElement.get( 0 ), "test element is focused after mousing down off the handle" );
```

Comment:
I don't like that the test is asserting the opposite of the expected behavior. When we finally move to WebDriver, this test would be correct, but for now, I'd prefer if we just mock `$.ui.safeBlur()` to verify the logic.

Would you mind updating the test to do that? If you need any help, just let us know.